From 55884cb1220d8953924c0bdf91a94f87db01c197 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Tue, 20 Dec 2022 15:22:17 +0800
Subject: [PATCH] Revert "Linux 5.10.160"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit a2428a8dcb4f3eb80e7d38dba0bf71e4ff20cecd.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6f7dae2f1a4e..bb9fab281555 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 160
+SUBLEVEL = 159
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

